mono/io-layer/sockets.c

```diff
@@ -1201,7 +1201,7 @@ WSAIoctl (guint32 fd, gint32 command,
 }
 
 #ifndef PLATFORM_PORT_PROVIDES_IOCTLSOCKET
-int ioctlsocket(guint32 fd, gint32 command, gpointer arg)
+int ioctlsocket(guint32 fd, unsigned long command, gpointer arg)
```

Comment:
That's a fair point. Now I'm curious on which value are you referring to.

git grep only revealed us using it with FIONREAD and FIONBIO. Both of which fit in a single byte.